lib/trackler/doc_file.rb

```diff
@@ -0,0 +1,92 @@
+class DocFile
+  DEFAULT_IMAGE_PATH = "/docs/img"
+
+  def self.find(basename:, track_dir:, image_path:)
+    dir = File.join(track_dir, "docs")
+
+    [
+      MarkdownFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+      OrgmodeFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+      NullFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+    ].detect(&:acceptable?)
```

Comment:
Using a default value for detect here:
`detect( -> { NullFile.new(basename: basename, docs_dir: dir, image_path: image_path) }, &:acceptable? )`
would remove the array ordering dependency on NullFile being listed last.
This may not be a problem if one trusts the unit tests.